test/regexs.test.ts

```diff
@@ -301,6 +301,11 @@ describe('regexps', () => {
             });
         });
     });
+    // The 'g' flag causes RegExp object to keep track of last match occurrence in `lastIndex`.
```

Comment:
I meant to test the effect of the `g` flag that prevented the correct usage of the regexp in the failed deploy, rather than just testing the definition :)